sql/dql/database-clients/connection-strings.md

```diff
@@ -49,25 +49,30 @@ dbprotocol://username:password
   %delim%property=value
 ```
 
-**dbprotocol** will be different for each database type. Postgres uses `postgres` and `pg`. MySQL uses `mysql`. Microsoft SQL Server uses `jdbc:sqlserver`.  
-**username:password** is a standard way of specifying authentication. For some databases (namely SQL server), the user and password are entered at the end, in the `property=value` section.  
-**hostname** is the address of the server the database is located on. If the database you are connecting to is _on your computer_, this value will be `localhost` or `0.0.0.0`. If it is _remote_, you will enter a hostname like `aws-us-east-1-portal.29.dblayer.com`.  
-**port** is how you know which port to connect to. The default port differs between database servers.  
-**delim** denotes a delimiter. Some databases read from the query string `?property=value&property=value`, some use a semicolon `/pokemon;property=value;property=value`. These properties and values are used to send extra information to the database.
-
-Here is the database connection string for the Postgres server hosting all of the examples from this course. You can connect to it, and perform those queries yourself!
-
-```
-postgres://ashketchum:iwannabetheverybest
-@aws-us-east-1-portal.29.dblayer.com
-  :23336/pokemon
-?ssl=true
-```
+- **dbprotocol** will be different for each database type:
+  - Postgres uses `postgres` and `pg`
+  - MySQL uses `mysql`
+  - Microsoft SQL Server uses `jdbc:sqlserver`.  
+- **username:password** is a standard way of specifying authentication
+  - for some databases (namely SQL server), the user and password are entered at the end, in the `property=value` section.  
+- **hostname** is the address of the server the database is located on
+  - if the database you are connecting to is _on your computer_, this value will be `localhost` or `0.0.0.0`
+  - if it is _remote_, you will enter a hostname like `aws-us-east-1-portal.29.dblayer.com`.  
+- **port** is how you know which port to connect to
+  - the default port differs between database servers.  
+- **delim** denotes a delimiter
+  - some databases read from the query string `?property=value&property=value`
+  - some use a semicolon `/pokemon;property=value;property=value`
+  - these properties and values are used to send extra information to the database.
 
 ---
 ## Practice
 
-Connect to a different `pokemon` database, this one hosted on MySQL. Use the username `gary`, password `rival`, to our database on `aws-us-west-1-portal.29.dblayer.com`. The port is `23336`.
+Connect to a `pokemon` database, that is hosted on MySQL. 
+
+Use the username `gary`, password `rival`, for the database on `aws-us-west-1-portal.29.dblayer.com`. 
```

Comment:
Since we already tell them which answers to use, the only unknown in this question is the order in which to use them, so let's remove the wrong answers as they are unnecessary?